Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.md

```diff
@@ -87,3 +87,4 @@
 | PH2134  | Set properties in any order                  | Getting other properties in a setter makes this setter dependent on the order in which these properties are set. |
 | PH2135  | Match namespace and Assembly Name            | The Assembly Name and Namespace must match. |
 | [PH2136](../Documentation/Diagnostics/PH2136.md)  | Avoid duplicate strings                      | Duplicate strings are less maintainable. |
+| PH2138  | Avoid void returns                           | Methods that return void are mysterious. |
```

Comment:
Please create an extended documentation MD file (PH2138.md)

Reply:
Updated this file to include the link. Created boilerplate PH2138.md. Filed Issue for Rodolfo to complete documentation.